builders.py

```diff
@@ -87,9 +87,7 @@ def start(self):
         echo "Checking for simplejson or json";
         "$VEPYTHON" -c 'import json' 2>/dev/null || "$VEPYTHON" -c 'import simplejson' ||
                     "$VE/bin/pip" install --no-index --download-cache="$PWD/.." --find-links="$PKG_URL" simplejson || exit 1;
-        echo "Checking for sqlite3, including pysqlite3 on Python 2.5";
         "$VEPYTHON" -c 'import sqlite3, sys; assert sys.version_info >= (2,6)' 2>/dev/null ||
-                    "$VEPYTHON" -c 'import pysqlite2.dbapi2' ||
                     "$VE/bin/pip" install --no-index --download-cache="$PWD/.." --find-links="$PKG_URL" pysqlite || exit 1
```

Comment:
If I'm reading this shell right, it: checks if the python version is 2.6 or higher, if that fails it tries to import `pysqlite2.dbapi2`, if that fails then it installs via pip. The first check is no longer going to fail now so the import check and pip install lines can be removed.